packages/lit-dev-content/site/docs/index.md

```diff
@@ -4,6 +4,8 @@ eleventyNavigation:
   key: What is Lit?
   parent: Introduction
   order: 1
+versionLinks:
+  v1:
```

Comment:
intentional? if so, does `/` work?

Reply:
We're doing effectively a naive `rootPath + '/' + versionLink` concat in the eleventy template here ,so using `/` here would make a double slash. I could also leave it off entirely, since it always defaults to the first page (which in both cases are "What is X"), but I figured this was kinda an explicit signal that I looked for the right matching path here.